node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+
+const messageText = process.argv[2] + '\n';
+
+const data = new Uint8Array(Buffer.from(messageText));
+fs.writeFile('note.txt', data, err => {
```

Comment:
If you add the encoding argument to `fs.writeFile` you don't need to encode it yourself.

```JAVASCRIPT
fs.writeFile('note.txt', messageText, 'utf8', err => {
```